doc/book/admin/upgrades.rst

```diff
@@ -58,95 +50,78 @@ How to upgrade from Tarantool 1.7 to 2.x
-under 1.7 for a while. It also renames a few configuration parameters, but old
-parameters are supported. The full list of breaking changes is available in
-`release notes for Tarantool 1.7 <https://github.com/tarantool/tarantool/releases>`_.
+..  important::
 
-1. Check with application developers whether application files need to be
-   updated due to incompatible changes (see
-   `1.7 release notes <https://github.com/tarantool/tarantool/releases>`_).
-   If yes, back up the old application files.
+    The only way to upgrade Tarantool from 1.6 to 2.x **without downtime** is
+    taking an intermediate step by upgrading from 1.6 to 1.10 and then to 2.x.
+    This also applies to upgrading Tarantool from 1.7 and 1.9.
 
-2. Stop the Tarantool server.
+    Before upgrading Tarantool from 1.x to 2.y, please read about the associated
+    :ref:`caveats <admin-upgrades-1.6-1.10>`.
 
-3. Make a copy of all data (see an appropriate hot backup procedure in
-   :ref:`Backups <admin-backups>`) and the package from which the current (old)
-   version was installed (for rollback purposes).
-
-4. Update the Tarantool server. See installation instructions at Tarantool
-   `download page <http://tarantool.org/download.html>`_.
+Before the upgrade
+~~~~~~~~~~~~~~~~~~
 
-5. Update the Tarantool database. Put the request ``box.schema.upgrade()``
-   inside a :doc:`box.once() </reference/reference_lua/box_once>` function in your Tarantool
-   :ref:`initialization file <index-init_label>`.
-   On startup, this will create new system spaces, update data type names (e.g.
-   num -> unsigned, str -> string) and options in Tarantool system spaces.
+#. Check the replica set health by running the following code on every replica:
 
-6. Update application files, if needed.
+   ..  code-block:: tarantoolsession
 
-7. Launch the updated Tarantool server using ``tarantoolctl`` or ``systemctl``.
+       box.info.ro -- shouldn't be "orphan"
+       box.info.ro_reason -- since 2.10, shouldn't be "orphan"
+       box.info.status -- should be "running"
 
-.. _admin-upgrades_replication_cluster:
+   If ``box.info.ro``, ``box.info.ro_reason``, or ``box.info.status`` have the value ``orphan``,
+   first resolve the replication issues and only then continue.
 
---------------------------------------------------------------------------------
-Upgrading Tarantool in a replication cluster
---------------------------------------------------------------------------------
+   If you're running Cartridge, you can also check node health in the UI.
 
-Tarantool 1.7 can work as a :ref:`replica <replication-architecture>`
-for Tarantool 1.6 and vice versa. Replicas
-perform capability negotiation on handshake, and new 1.7 replication features
-are not used with 1.6 replicas. This allows upgrading clustered configurations.
+#. Make sure each replica connected to the rest of the replica set.
+   Run the following, replacing the ``id`` with the replica ID (the output of ``box.info.id``):
```

Comment:
I think it would be simpler to run
`box.info.replication`, and then check the output table (which is indexed by replica ids and contains upstream, downstream for each id). Each upstream and downstream should have status = follow. The only exceptions are upstream and downstream belonging to this particular instance (they are indexed by `box.info.id`)